site/src/components/Input/Input.tsx

```diff
@@ -0,0 +1,29 @@
+/**
+ * Copied from shadc/ui on 11/13/2024
+ * @see {@link https://ui.shadcn.com/docs/components/input}
+ */
+import * as React from "react";
```

Comment:
seems like there's a lot of bouncing back and forth between `* as React` and actually importing things by name